### PR TITLE
Change colors

### PR DESCRIPTION
**Pull request by GresosJ**
Before approve this, let the design team see it first

### CODE REVIEW

**Comment by ruilopesm** [comment]
image

CeSIUM's logo should be the light one 🙏🏼

**Comment by GresosJ** [comment]
The design team already saw it, and we can continue we this color scheme.
I just need the white CeSIUM logo version

File: src/data/blog.json [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
...lias eaque debitis laboriosam nam voluptatum a asperiores modi nulla expedita eligendi quas suscipit, consequatur ipsum quos enim eum velit illo. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit alias eaque debitis laboriosam nam voluptatum a asperiores modi nulla expedita eligendi quas suscipit, consequatur ipsum quos enim eum velit illo.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit alias eaque debitis laboriosam nam voluptatum a asperiores modi nulla expedita eligendi quas suscipit, consequatur ipsum quos enim eum velit illo. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit alias eaque debitis laboriosam nam voluptatum a asperiores modi nulla expedita eligendi quas suscipit, consequatur ipsum quos enim eum velit illo. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit alias eaque debitis laboriosam nam voluptatum a asperiores modi nulla expedita eligendi quas suscipit, consequatur ipsum quos enim eum velit illo. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit alias eaque debitis laboriosam nam voluptatum a asperiores modi nulla expedita eligendi quas suscipit, consequatur ipsum quos enim eum velit illo."
```
Comment (MarioRodrigues10) [inline src/data/blog.json:7]: Why you need to increase the Lorem ipsum text?

Comment (GresosJ) [reply src/data/blog.json:7]: I was testing with big text's how will look the papers page

**Comment by GresosJ** [comment]
Now I thing everything looks good for me

**Review by tiago-bacelar — CHANGES_REQUESTED** [review]
There's a white line on top of the page in small mode (not extra small, just small). I looked a bit into it but didn't find any obvious error, so I'll leave it to you :)

Also, I think there isn't enough contrast between the logos and the background, especially elsa's.

**Comment by tiago-bacelar** [comment]
Nevermind, the cache just refreshed. Elsa's logo is fine. The others are still a bit dark though...

**Review by tiago-bacelar — APPROVED** [review]
Still think the company logos could be lighter...

**Pull request merged**